docs/FilterList.md

```diff
@@ -185,3 +185,65 @@ const CustomerList = props => (
 {% endraw %}
 
 **Tip**: The `<FilterList>` Sidebar is not a good UI for small screens. You can choose to hide it on small screens (as in the previous example). A good tradeoff is to use `<FilterList>` on large screens, and the Filter Button/Form combo on Mobile.
+
+## Customize How Filters Are Applied
```

Comment:
I'd call it "cumulative filters"

Reply:
I disagree, that's only one of the possible use cases

Reply:
@djhi Can you still mention the words "cumulative filters" in the doc, before the example, so that an Algolia search would still lead to this page?

Reply:
Cumulative, additive. I'm not sure what's the best one here